rock_paper_scissor.py

```diff
@@ -51,17 +52,18 @@ def round_win(player1, player2, player1_score, player2_score):
     elif player1==3 and player2==2:
         player1_score+=1
         print("Player 1 wins the round")
-    elif (player1==1 and player2==1) or (player1==2 and player2==2) or (player1==3 and player2==3):
+    elif (player1==1 and player2==1) or (player1==2 and player2==2) 
+    or (player1==3 and player2==3):
         print("This round is a draw !!!")
     return player1_score, player2_score
 
 
 def winner(player1_score, player2_score):
     if player1_score > player2_score:
-        print("---------------")
+        print("#"*128)
         return "!!! PLAYER 1 WON THE GAME !!!"
     elif player1_score < player2_score:
-        print("---------------")
+        print("#"*128)
         return "!!! COMPUTER WON THE GAME !!!"
     else:
         print("---------------")

```

Comment:
You missed a line here. These can all be placed to the top of the function as well, like so:
```py
def winner(player1_score, player2_score):
   print("#"*128)
```
Then it doesn't have to be repeated 3 times for each conditional.

Reply:
I also think 128 is too excessive for the number of pound signs shown. The original snippet had 15 hyphens.

Reply:
Hey @Pepper-Wood . Thanks for pointing this out. I'll get this check by @mehta-raghav 👍

Reply:
Hey, it seems my PEP8 changes didn't commit through. I've done it now. Please re-evaluate. Thanks and sorry for the inconvenience.
As for the 128 pound sign I went for a horizontal rule across the console. If its a bit much I'll revert it back.

@Pepper-Wood thanks for pointing this out.